render/render.go

```diff
@@ -72,6 +73,33 @@ func Render(app *types.App, env map[string]string) (*composetypes.Config, error)
 		}
 		renderers = rl
 	}
+	return renderers, nil
+}
+
+// RenderConfig renders an arbitrary string
+func RenderConfig(app *types.App, env map[string]string, config string) (string, error) { //nolint:golint
```

Comment:
Why is `//nolint:golint` needed? 🤔

Reply:
Because golint complains about stutter since the package name is render. But I don't like naming this just 'Config'.

Reply:
we should follow golang naming conventions (eg: having render.Config, render.Attachment etc.)